xds/internal/xdsclient/xdsresource/unmarshal_lds_test.go

```diff
@@ -1719,6 +1721,63 @@ func (s) TestUnmarshalListener_ServerSide(t *testing.T) {
 	}
 }
 
+func TestRbacNamePassthrough(t *testing.T) {
+	filters := []*v3httppb.HttpFilter{
+		{
+			Name: "StdoutAuditLogger",
+			ConfigType: &v3httppb.HttpFilter_TypedConfig{
+				TypedConfig: testutils.MarshalAny(&v3rbacpb.RBAC{
+					Rules: &v3baserbacpb.RBAC{
+						Action: v3baserbacpb.RBAC_DENY,
+
+						Policies: map[string]*v3baserbacpb.Policy{
+							"authz_deny_policy_1": {
+								Principals: []*v3baserbacpb.Principal{
+									{Identifier: &v3baserbacpb.Principal_OrIds{OrIds: &v3baserbacpb.Principal_Set{
+										Ids: []*v3baserbacpb.Principal{
+											{Identifier: &v3baserbacpb.Principal_Authenticated_{
+												Authenticated: &v3baserbacpb.Principal_Authenticated{PrincipalName: &v3matcherpb.StringMatcher{
+													MatchPattern: &v3matcherpb.StringMatcher_Exact{Exact: "spiffe://foo.abc"},
+												}},
+											}},
+										},
+									}}},
+								},
+								Permissions: []*v3baserbacpb.Permission{
+									{Rule: &v3baserbacpb.Permission_Any{Any: true}},
+								},
+							},
+						},
+						AuditLoggingOptions: &v3baserbacpb.RBAC_AuditLoggingOptions{
+							AuditCondition: v3baserbacpb.RBAC_AuditLoggingOptions_ON_DENY,
+							LoggerConfigs: []*v3baserbacpb.RBAC_AuditLoggingOptions_AuditLoggerConfig{
+								{AuditLogger: &v3corepb.TypedExtensionConfig{Name: "stdout_logger", TypedConfig: testutils.MarshalAny(&v3auditloggersstreampb.StdoutAuditLog{})},
```

Comment:
nit: if the struct wrapped by the `{` is not a single liner, prefer to break lines, following the existing style in the file, i.e.
```Go
{
  AuditLogger: &v3corepb.TypedExtensionConfig{
    Name: "..",
    TypedConfig: ...,
  },
}
```

It will be easier for readers to tell which field belongs to which struct.

Reply:
Yeah I should've done this in the first place, thanks for the comment

This is already a mess of hugely nested structures, so anything to make it more manageable for human consumption :)